src/FffPolygonGenerator.cpp

```diff
@@ -486,6 +527,14 @@ void FffPolygonGenerator::processDerivedWallsSkinInfill(SliceMeshStorage& mesh)
     }
 }
 
+/*
+ * This function is executed in a parallel region based on layer_nr.
+ * When modifying make sure any changes does not introduce data races.
+ *
+ * generateInsets only reads and writes data for the current layer
```

Comment:
This documentation reports on some `generateInsets` function. Probably a copy-paste mistake. I'm removing it in 6861020c6b1282a9db28f4c3eeea460ff9651245.